tools/src/icon4pytools/py2fgen/wrappers/diffusion_wrapper.py

```diff
@@ -335,4 +341,41 @@ def grid_init(
         v2c=v2c,
         e2c2v=e2c2v,
         c2v=c2v,
+        grid_id="icon_grid",
```

Comment:
This one in general is a uuid, which is available in ICON from `t_patch%grid_uuid` ( `mo_model_domain.f90`) in case you care. We don't use it really for any specific thing.

Reply:
Since this `uuid` doesn't have any functional value I would prefer to not introduce an additional argument for that in the wrapper.